husky_bringup/launch/um6_config/um6.launch

```diff
@@ -22,7 +22,7 @@
 
   <!-- Filter raw gyro, accel and mag data into a usable orientation -->
   <node pkg="nodelet" type="nodelet" name="imu_filter" args="load imu_filter_madgwick/ImuFilterNodelet imu_manager">
-    <rosparam file="$(env HUSKY_MAG_CONFIG)" />
+    <rosparam file="$(optenv HUSKY_MAG_CONFIG config/mag_config_default.yaml)" />
```

Comment:
Is this now package-relative by default? Historically it was relative to the cwd of wherever roslaunch was invoked from, which was.... basically useless (that said, `$(eval)` now gives better options regardless— either way, the env-hook is no longer necessary).

Reply:
Appears to be package-relative.  Worked in kinetic

Reply:
Cool, as long as it still does the right thing when invoked from a systemd job or whatever.

Reply:
Same concern but if it works great!

Reply:
Was fixed properly in https://github.com/husky/husky/pull/140.